cpu/efm32/families/efm32gg/efm32-info.mk

```diff
@@ -0,0 +1,47 @@
+# This file is automatically generated, and should not be changed. There is
```

Comment:
@basilfx I have a THUNDERBOARD EFM32GG12 here, for whose MCU there is no entry in this file. Is this file really created automatically and if so, how. Or does the entry have to be added manually?

Reply:
This file is generated using https://github.com/basilfx/EFM2Riot. That script parses the Gecko SDK and generates the necessary files. I'll admit that it's a bit outdated now.